src/main/scala/abalone/format/Uci.scala

```diff
@@ -24,7 +24,7 @@ object Uci {
       promotion: Option[PromotableRole] = None
   ) extends Uci {
 
-    def keys = orig.key + dest.key
+    def keys = orig.officialNotationKey + dest.officialNotationKey
```

Comment:
So this should be left as proper uci. Not Abalone format.
Our use of formats/notation is a bit inconsistent across strategygames, but `Uci` is always uci. We've tended to use lila to convert our backend format (usually uci) into the correct notation for display